hypervisor/dm/vpci/pci_pt.c

```diff
@@ -195,6 +195,31 @@ void vdev_pt_write_vbar(struct pci_vdev *vdev, uint32_t idx, uint32_t val)
 	}
 }
 
+void vdev_pt_restore_bars(const struct pci_vdev *vdev)
```

Comment:
Do we need this kind of complexity? I thought we should have an API of reset_pcie_dev... Inside this API, we save the BARs and then try to use whatever RESET method, and then restore BARs.

This way, we don't need to care MEM64 or 32 bars.

Reply:
More specifically, it enforces: decouple of P and V PCIe devices.

Reply:
OK.